text/latex.go

```diff
@@ -18,15 +18,17 @@ import (
 )
 
 // Latex parses, formats and renders LaTeX.
-type Latex struct{}
+type Latex struct {
+	DPI float64
```

Comment:
Add a doc comment that this sets that LaTeX font resolution and also that resolution defaults to 72 if this is unset.

Reply:
done